threshold_encrpytion.go

```diff
@@ -0,0 +1,43 @@
+package cleisthenes
+
+import "github.com/boohyunsik/tpke"
+
+type Config struct {
+	threshold   int
+	participant int
+}
+
+type KeySet struct {
+	pkSet *tpke.PublicKeySet
+	skSet *tpke.SecretKeySet
+}
+
+// Setup makes public key set and secret key set.
+func Setup(c Config) *KeySet {
+	secretKeySet := tpke.RandomSecretKeySet(c.threshold)
+	publicKeySet := secretKeySet.PublicKeySet()
+	return &KeySet{
+		pkSet: publicKeySet,
+		skSet: secretKeySet,
+	}
+}
+
+// Encrypt encrypts some byte array message.
+// Length of byte array must be shorter than 32 bytes. (TODO : fix it.)
```

Comment:
Does `TODO` plan to fix the `Length of byte array must be shorter than 32 bytes.`?